### PR TITLE
Added month misspells

### PR DESCRIPTION
**Pull request by Paradact**
Added misspellings of several months

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -23546,6 +23548,7 @@ ojective->objective
 ojects->objects
 ojekts->objects
 okat->okay
+oktober->October
```
[the remark this answers is not in the document]
Comment (Paradact) [reply codespell_lib/data/dictionary.txt:23557]: Thank you for your suggestions!

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -22334,6 +22341,7 @@ mystrow->maestro
 mystrows->maestros
 Mythraic->Mithraic
 myu->my
+märz->March
```
[the remark this answers is not in the document]
Comment (peternewman) [reply codespell_lib/data/dictionary.txt:22344]: I wonder if they should all move into another dictionary foreign->en, so people checking multi-lingual text can do so safely by not using that dictionary.

Well perhaps not the ones that aren't valid words as the typo.

More generally I wonder if we need another dictionary there too, if you change attache->attaché I could imagine having a non-ASCII character in a variable name could upset some programming languages and gardai->gardaí could cause confusion in lots where it looks the same but acts differently!

**Review by DimitriPapadopoulos — APPROVED** [review]

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -20920,6 +20925,7 @@ magnatude->magnitude
 magnificient->magnificent
 magolia->magnolia
 mahcine->machine
+mai->May
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:20928]: Also `mai` may be `mail`, `main` or simply `may`.

**Review by DimitriPapadopoulos — COMMENTED** [review]
I think we can merge once `mai` is addressed.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by skangas** [comment]
LGTM

**Review by skangas — APPROVED** [review]

**Pull request merged**